### PR TITLE
Fix #151

### LINKED ISSUE #151: Wrong unit test causes CI to fail
```
--- FAIL: TestRuleNamesResponse (0.00s)
    strings.go:50: 
        	Error Trace:	strings.go:50
        	            				http.go:124
        	            				http.go:89
        	            				http.go:110
        	            				server_test.go:372
        	            				timeout.go:33
        	            				asm_amd64.s:1357
        	Error:      	Not equal: 
        	            	expected: map[string]interface {}{"rules":[]interface {}{"foo.rules.internal.bar", "ccx_rules_ocp.external.rules.node_installer_degraded"}, "status":"ok"}
        	            	actual  : map[string]interface {}{"rules":[]interface {}{"ccx_rules_ocp.external.rules.node_installer_degraded", "foo.rules.internal.bar"}, "status":"ok"}
      	            	
        	            	Diff:
        	            	--- Expected
        	            	+++ Actual
        	            	@@ -2,4 +2,4 @@
        	            	  (string) (len=5) "rules": ([]interface {}) (len=2) {
        	            	-  (string) (len=22) "foo.rules.internal.bar",
        	            	-  (string) (len=52) "ccx_rules_ocp.external.rules.node_installer_degraded"
        	            	+  (string) (len=52) "ccx_rules_ocp.external.rules.node_installer_degraded",
        	            	+  (string) (len=22) "foo.rules.internal.bar"
        	            	  },
        	Test:       	TestRuleNamesResponse
        	Messages:   	{"rules": ["foo.rules.internal.bar", "ccx_rules_ocp.external.rules.node_installer_degraded"],"status": "ok"}
        	            	and
        	            	{"rules":["ccx_rules_ocp.external.rules.node_installer_degraded","foo.rules.internal.bar"],"status":"ok"}
        	            	should represent the same json
FAIL
```

### PR DESCRIPTION
**Pull request by joselsegura**
# Description

The problem is caused because the list returned by the `rule_ids` endpoint  is generated from a `map`, so its order is not granted.
This fix orders the output using `sort`, making the result predictable

Fixes #151 

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## Testing steps

Regular CI

### CODE REVIEW

File: content/content.go [hunk 1]
```diff
@@ -121,6 +122,7 @@ func (s *RulesWithContentStorage) GetRuleIDs() []string {
 		ruleIDs = append(ruleIDs, ruleContent.Plugin.PythonModule)
 	}
 
+	sort.Strings(ruleIDs)
```
Comment (Sergey1011010) [inline content/content.go:125]: I think it's better to sort when comparing in tests

Comment (joselsegura) [reply content/content.go]: I think about it and you are right. I just modified it

File: server/server_test.go [hunk 2]
```diff
@@ -390,8 +392,32 @@ func TestRuleNamesResponse(t *testing.T) {
...
+	}
+
+	if err := json.Unmarshal(got, &gotResp); err != nil {
+		err = fmt.Errorf(`"got" is not JSON. value = "%v", err = "%v"`, got, err)
+		helpers.FailOnError(t, err)
+	}
+
+	sort.Strings(expectedResp.Rules)
+	sort.Strings(gotResp.Rules)
+	assert.Equal(t, expectedResp, gotResp)
```
Comment (Sergey1011010) [inline server/server_test.go:422]: btw, there some function in assert package which does the same as these 3 lines

Comment (joselsegura) [reply server/server_test.go:422]: Mmm, interesting. I discovered a lot of new asserting functions on `assert` package, but I cannot find the one you are referring to.

The most likely for me is using 2 `assert.Subset` (`expectedResp` is a subset of `gotResp` and vice versa).

Comment (joselsegura) [reply server/server_test.go:422]: `assert.ElementsMatch` is the answer. Nothing like writting a comment asking for something to find it by yourself

Comment (joselsegura) [reply server/server_test.go]: Updated commit using `assert.ElementsMatch` instead of sorting by myself

**Pull request merged**